src/backend/executor/execTuples.c

```diff
@@ -1596,6 +1596,14 @@ slot_getsysattr(TupleTableSlot *slot, int attnum, Datum *result, bool *isnull)
                         case TableOidAttributeNumber:
 							*result = ObjectIdGetDatum(slot->tts_tableOid);
 							break;
+						case MinTransactionIdAttributeNumber:
+						case MaxTransactionIdAttributeNumber:
+						case MinCommandIdAttributeNumber:
+						case MaxCommandIdAttributeNumber:
+							if (slot->PRIVATE_tts_flags & TTS_APPEND_OPTIMIZED) {
+								elog(ERROR, "AO-table has no xmin, xmax, cmin, cmax, attributes");
+							}
```

Comment:
braces are redundant here

Reply:
Yes, braces are not strictly necessary here but useful for readability and proof against accidental change inside condition.

Reply:
opened brace (place) is not correspond to pgindent!